docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -110,7 +101,6 @@ You can set the location of the compiler with environment variables. Use the pat
 
 | Toolchain | Environment variable |
 | --------- | --------- |
-| Arm Compiler 5 | `MBED_ARM_PATH` |
 | Arm Compiler 6 | `MBED_ARMC6_PATH` |
 | IAR EWARM Compiler | `MBED_IAR_PATH` |
```

Comment:
Note IAR is not supported in Mbed OS 6.0, therefore everything related to IAR should be removed.